docroot/modules/humsci/hs_dashboard/hs_dashboard.info.yml

```
@@ -0,0 +1,8 @@
+name: 'H&S Dashboard'
+type: module
+description: 'Custom code and modifications for H&S Dashboards'
+core_version_requirement: ^10.3 || ^11
+package: 'Humanities & Sciences'
+dependencies:
+  - stanford_migrate
+  - hs_capx
```

```suggestion
  - stanford_migrate
  - hs_capx
  - dashboard
```

I'd add the dashboard module as a dependency.